ocp_resources/virtual_machine_export.py

```diff
@@ -21,46 +21,57 @@ class SourceKind:
 
     def __init__(
         self,
-        name=None,
-        namespace=None,
-        client=None,
-        teardown=True,
-        token_secret_ref=None,
-        source_api_group=None,
-        source_kind=None,
-        source_name=None,
-        timeout=TIMEOUT_1MINUTE,
-        delete_timeout=TIMEOUT_1MINUTE,
-        yaml_file=None,
+        source_api_group: str | None = None,
+        source_kind: str | None = None,
+        source_name: str | None = None,
+        token_secret_ref: str | None = None,
+        ttl_duration: str | None = None,
+        delete_timeout: int = TIMEOUT_1MINUTE,
         **kwargs,
-    ):
+    ) -> None:
```

Comment:
Should this file be regenerated using the class generator script?

Reply:
With class generator it expects the whole `source` dict instead of `source_api_group`,  `source_kind`, `source_name` and removes the `SourceKind` class.

If it's a requirement, I can do this, but currently looks much more convenient.